### PR TITLE
Adds 'cell_noskill' which blocks skill usage.

### PR DESCRIPTION
**Pull request by bWolfie**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)
Adds the `cell_noskill` cell, which allows the setting of cells which players cannot cast skills.

**Affected Branches:** 

[//]: # (Master? Slave?)
- [x] master
- [x] stable

**Issues addressed:**

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2188)

File: db/constants.conf [hunk 1]
```diff
@@ -434,6 +434,9 @@ constants_db: {
 	cell_landprotector: 5
 	cell_novending:     6
 	cell_nochat:        7
+	//cell_icewall:       8
```
Comment (4144) [inline db/constants.conf]: why this constants commented?

Comment (EyesOfAHawk) [reply db/constants.conf]: they weren't there before, iassumed there was a reason for it? i didnt know what to do with them so i just added them as comments.

**Comment by EyesOfAHawk** [comment]
just had a thought. Is constants.md file auto-updated, or should I manually add these changes to it?

**Pull request merged**